cilantro_audit/home_page.py

```diff
@@ -41,7 +46,7 @@ def build(self):
         sm.add_widget(AdminPage(name=ADMIN_SCREEN))
         sm.add_widget(AuditorPage(name=AUDITOR_SCREEN))
         sm.add_widget(CreateAuditTemplatePage(name=CREATE_AUDIT_TEMPLATE_PAGE))
-        sm.add_widget(CompletedAuditsListPage(name=COMPLETED_AUDITS_LIST_PAGE))
```

Comment:
Why did you delete this line? Now it raises an exception when trying to go to this page.

Reply:
Looks like these were from the merge with my branch, which I had started before these were merged into master. Thanks for catching that!